carbonmark/pages/projects/index.tsx

```diff
@@ -22,17 +22,21 @@ export interface ProjectsPageStaticProps {
   vintages: Vintage[];
 }
 
-export const getStaticProps: GetStaticProps<ProjectsPageStaticProps> = async (
-  ctx
-) => {
+export const getServerSideProps: GetServerSideProps<
+  ProjectsPageStaticProps
+> = async (ctx) => {
   try {
-    const projects = await getProjects();
-    const vintages = await getVintages();
-    const countries = await getCountries();
-    /** @note because the API is returning trailing empty spaces on some categories, trim them here */
-    const categories = (await getCategories()).map(update("id", trim));
-
-    const translation = await loadTranslation(ctx.locale);
+    const [projects, vintages, countries, categories, translation] =
+      await Promise.all([
+        fetchProjects(ctx.query),
+        getVintages(),
+        getCountries(),
+        /** @note because the API is returning trailing empty spaces on some categories, trim them here */
+        getCategories().then((categories) =>
+          categories.map(update("id", trim))
+        ),
```

Comment:
We can remove the comment, empty strings are no longer a problem.
This can become
```
.then(({ id }) => id)
```